open-sphere-base/core/src/main/java/io/opensphere/core/map/MapManagerMenuProvider.java

```diff
@@ -64,46 +87,14 @@
                 center.addActionListener(arg0 -> new ViewerAnimator(myMapManager.getStandardViewer(), pos).start());
                 menuItems.add(center);
 
-                JMenuItem copyToClipboard = new JMenuItem("Copy coordinates to clipboard",
-                        new GenericFontIcon(AwesomeIconSolid.COPY, Color.WHITE));
-                copyToClipboard.addActionListener(arg0 ->
-                {
-                    if (myUnitsRegistry != null)
-                    {
-                        Class<? extends Angle> angleUnits = myUnitsRegistry.getPreferredUnits(Angle.class);
-                        String label = pos.toDisplayString(angleUnits, (Class<? extends Length>)null);
-
-                        try
-                        {
-                            Toolkit.getDefaultToolkit().getSystemClipboard().setContents(new StringSelection(label), null);
-                            Notify.info("Copied " + label + " to clipboard");
-                        }
-                        catch (IllegalStateException ex)
-                        {
-                            JOptionPane.showMessageDialog(null, "Failed to copy to clipboard.");
-                        }
-                    }
-                });
+                JMenuItem copyToClipboard = new JMenuItem("Open Coordinate Viewer",
+                        new GenericFontIcon(AwesomeIconSolid.MAP_MARKER, Color.WHITE));
+
+                JDialog theCordMenu = CordMenu(pos);
```

Comment:
General convention- don't start variable names with "the"

Reply:
Changed

Reply:
fixed